python/cuml/test/test_umap.py

```diff
@@ -364,3 +365,25 @@ def test_umap_transform_trustworthiness_with_consistency_enabled():
     embedding = model.transform(transform_data, convert_dtype=True)
     trust = trustworthiness(transform_data, embedding, 10)
     assert trust >= 0.92
+
+
+@pytest.mark.parametrize('n_neighbors', [5, 15])
+def test_umap_knn_parameters(n_neighbors):
+    data, labels = datasets.make_blobs(
+        n_samples=2000, n_features=10, centers=5, random_state=0)
+    data = data.astype(np.float32)
+
+    knn = NearestNeighbors(n_neighbors=n_neighbors)
```

Comment:
A good verification that this does what we want and nothing further would be to build two models using the consistent codepath and compare their resulting embeddings directly- modelA is built w/ X only and modelB is built with X, `knn_indices`, and `knn_dists`.

Reply:
Unfortunately, for some reasons, the `NearestNeighbors` model does not produce the same distance values as the ones produced by `brute_force_knn` used by UMAP. Will have to investigate that, but, this specific test might not be ready for burndown.

Reply:
@wxbn were you able to determine why there are discrepancies here?